### PR TITLE
removed vs_ref stat_file and unused parameter and the duplicate stat_…

### PR DESCRIPTION
**Pull request by jasonmotha**
…file entry

#### Checklist
- Have you updated the README (yes/no)?
- Have you updated the CHANGELOG (yes/no)?

### CODE REVIEW

**Comment by jasonmotha** [comment]
I found recently that stat_file was stored in both root_params and sim_params.

This PR removes the one in sim_params as it is a copy of the one in root params. And tidied up a couple of things.

File: workflow/automation/tests/test_shared_workflow/test_install_shared.py [hunk 1]
```diff
@@ -23,6 +23,7 @@ def test_install_simulation(set_up):
 
         # The pickle file was missing the 10th argument. This enables the vm to be checked.
         input_params.insert(10, True)
+        input_params.pop(9)  # Removes the vs_ref stat file parameter
```
Comment (jpa198) [inline workflow/automation/tests/test_shared_workflow/test_install_shared.py:26]: Lol

**Review by jpa198 — APPROVED** [review]

File: workflow/automation/tests/test_shared_workflow/test_install_shared.py (hunk 1 above)
Comment (sungeunbae) [reply workflow/automation/tests/test_shared_workflow/test_install_shared.py:26]: Can we fix the pickle file? It's getting multiple band-aids.

**Review by sungeunbae — CHANGES_REQUESTED** [review]

File: workflow/automation/tests/test_shared_workflow/test_install_shared.py (hunk 1 above)
Comment (jpa198) [reply workflow/automation/tests/test_shared_workflow/test_install_shared.py:26]: The test data is quite a mission to change at the moment. The current process involves logging into the test machine (I don't think I even have a working account on it), finding the right data, loading the data from pickle, editing it, saving back into pickle and running a test to make sure it works. Which is a lot of effort compared with adding a line or two to patch the test.
In the short term a few more careful patches won't hurt, however in the longer term I think testing needs a relatively large overhaul.

Comment (jasonmotha) [reply workflow/automation/tests/test_shared_workflow/test_install_shared.py:26]: @sungeunbae Thoughts?

**Review by sungeunbae — APPROVED** [review]

**Pull request merged**